cmd/open.go

```diff
@@ -114,5 +115,22 @@ func (o *openOptions) run(cmd *cobra.Command) error {
 	sessionURI := formatter.FormatSessionURI(o.serverURL, openResp.Payload.Path, o.token)
 	displayer.PrintColorable(sessionURI+"\n", cmd.OutOrStdout(), text.FgGreen)
 	cmd.Println("It could take several minutes to start the interactive session.")
+
+	infoParams := operations.NewInfoParams()
+	infoParams.SetAccessToken(o.token)
+	infoResp, err := api.Operations.Info(infoParams)
+	if err != nil {
+		return nil
+	}
+	maxInactivityDays := infoResp.Payload.MaximumInteractiveSessionInactivityPeriod
+	if maxInactivityDays != nil {
```

Comment:
```suggestion
	if maxInactivityDays != nil  && maxInactivityDays.Value != nil {
```
This will fail with `panic: runtime error: invalid memory address or nil pointer dereference` if `maximum_interactive_session_inactivity_period` is in the server's reply but `value` is `null`. Could you please add a testcase where this happens?